ninjalooter/models.py

```diff
@@ -7,9 +7,11 @@
 import dateutil.parser
 import wx
 
+
 from ninjalooter import config
 from ninjalooter import extra_data
 from ninjalooter import logger
+from ninjalooter import constants
```

Comment:
Imports are sorted alphabetically, so this would go right between config and extra_data